server/db/courses/courses.js

```diff
@@ -0,0 +1,71 @@
+/* Create Courses database collection */
+Courses = new Mongo.Collection('Courses');
+
+/* Create Courses database schema */
+courseSchema = new SimpleSchema({
+    name: {
+        type: String,
+        label: "Course Name"
+    },
+    number: {
+        type: Number,
+        label: "Course Number"
+    },
+    description: {
+        type: String, 
+        defaultValue: "",
+        label: "Course Description"
+    },
+    openInfo: {
+        type: [String],
+        label: "Customizable Course Information"
+    }
+});
+
+/* This function adds the passed in course to the collection if valid */
+function addCourse(newCourse) {
+    try {
+        courseSchema.validate(newCourse);
+        Courses.insert(newCourse);
+        console.log("Inserted one course.");
+    }
+    catch(err) {
+        console.log("The format of the course is wrong!");
+    } 
+}
+
+/* Insert Course example */
+/*
+const myCourse = {
+    name: 'Great Practical Ideas for Computer Scientists',
+    number: 15131,
+    description: 'This course focuses on teaching students how to use the popular tools for software development.',
+    openInfo: ["Homework:", "hw1", "Labs", "lab1"]
+};
+addCourse(myCourse);
+*/
+
+
+/* Get Course function */
+// var myId = "BNFc6X5Tc6x6PuMSu";
+
+/* Functions to get course data when given id */
+function getCourse(id) {
+    return Courses.findOne({_id: id});
```

Comment:
Let's not use findOne blindly-

1) Enforce unique in the Schema (https://github.com/aldeed/meteor-simple-schema)
2) Make sure to add to your Database Schema that the field is unique (this can be done with indices)